packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/useChartCartesianAxis.axisHighlight.test.tsx

```diff
@@ -181,4 +182,29 @@ describe('useChartCartesianAxis - axis highlight', () => {
       ]);
     },
   );
+
+  it.skipIf(isJSDOM)('should allow to highlight axes without data', async () => {
```

Comment:
Should we have a test case for a continuous scale as well? The implementation seems to be different for discrete and continuous scales.

Reply:
Good idea. I added a test for the x-axis which is a band scale :+1: